### PR TITLE
OrmElf.getInClausePlaceholdersForCount

### PR DESCRIPTION
**Pull request by sergiorussia**
@brettwooldridge, i use OrmElf.getInClausePlaceholders intensively but in many cases i only have array/list/set/whatever of integers, which for some reason forces to pass String varargs. here's a more efficient alternative which does not require varargs (and slightly more efficient in its impl)

### CODE REVIEW

File: src/main/java/com/zaxxer/sansorm/OrmElf.java [hunk 1]
```diff
@@ -67,13 +67,7 @@ private OrmElf() {
     * just be the conditional portion that would normally appear after the "WHERE", and therefore
     * the clause "WHERE" is automatically appended to the generated "SELECT .. FROM" SQL, followed
     * by the specified clause.  For example:<p>
-    * <code>
-    *    User user = OrmElf.objectFromClause(connection, User.class, "username=?", userName);
-    * </code>
-    * or<p>
-    * <code>
-    *    User user =
```
Comment (sergiorussia) [inline src/main/java/com/zaxxer/sansorm/OrmElf.java]: ??? :)

**Comment by sergiorussia** [comment]
@brettwooldridge any chances? there are more coming, waiting for this one first

**Comment by lfbayer** [comment]
Looks reasonable to me. I'm just assuming it works :)

**Comment by sergiorussia** [comment]
@lfbayer, tests are there to make you sure :)

**Comment by lfbayer** [comment]
Yes, the tests added to my confidence. It'd be nice if sansorm had better test coverage.

**Comment by sergiorussia** [comment]
(i'm sorry for messing up the master branch history a bit, all consequent PRs will be from separate branches)

**Pull request merged**